src/normed.jl

```diff
@@ -241,7 +266,8 @@ Base.Rational{Ti}(x::Normed) where {Ti <: Integer} = convert(Ti, reinterpret(x))
 Base.Rational(x::Normed) = reinterpret(x)//rawone(x)
 
 # Traits
-abs(x::Normed) = x
+abs(x::Normed{<:Unsigned}) = x
+abs(x::T) where T<:Normed  = T(abs(x.i), 0)
```

Comment:
Why does not `abs(::Normed{<:Signed})` return `Normed{<:Unsigned}`? (Of course, this is not a question.)

Reply:
Try this: `abs(Int8(-5))`. In general Julia doesn't change the number type, even to the point where `abs(typemin(Int8))` returns a negative number!

Reply:
That's right. Therefore, it can be concluded that `-(x :: N0f8, y :: N0f8) = S7f8 (Int16 (x.i)-Int16 (y.i), 0)` is not good! :stuck_out_tongue_closed_eyes:

Reply:
I agree, this is in conflict with the rest of Julia. I guess we could return `Normed{<:Unsigned}`, since the only other "always agrees with expectations" is to widen which also changes the type.